src/coreclr/nativeaot/Runtime/gcenv.ee.cpp

```diff
@@ -56,7 +56,7 @@ void GCToEEInterface::RestartEE(bool /*bFinishedGC*/)
 {
     FireEtwGCRestartEEBegin_V1(GetClrInstanceId());
 
-#if defined(TARGET_ARM) || defined(TARGET_ARM64)
+#if defined(TARGET_ARM) || defined(TARGET_ARM64) || defined(TARGET_LOONGARCH64)
```

Comment:
```suggestion
#if !defined(TARGET_X86) || !defined(TARGET_AMD64)
```
The weak memory ordered processors should be the default. Also, could you please update `#endif` to match?

Reply:
OK. Thanks!